etc/proto/run.sh

```diff
@@ -17,14 +18,14 @@ fi
 
 go get github.com/gogo/protobuf/protoc-gen-gogofast
 
-cd ${GOPATH}/src/github.com/pachyderm/pachyderm
-for i in $(find src -name "*.proto"); do \
+cd "${GOPATH}/src/github.com/pachyderm/pachyderm"
+for i in src/**/*.proto; do \
     if ! grep -q 'go_package' "${i}"; then
         echo -e "\e[1;31mError:\e[0m missing \"go_package\" declaration in ${i}" >/dev/stderr
```

Comment:
We might want an `|| true` appended here, since we set -e and I am expecting we want to continue iterating through the loop on errors

Reply:
This is for the `grep` command, right? I actually assume the `if` would capture the exit code such that `set -e` wouldn't matter, but let me double-check.

Reply:
Yeah, this is safe / will print `ok`:

```bash
#!/bin/bash
set -e
if ! which foobar; then
    echo "ok"
fi
```